website/docs/installation/docker-compose.mdx

```diff
@@ -12,32 +12,74 @@ This installation method is for test-setups and small-scale production setups.
+<iframe
+    width="560"
+    height="315"
+    src="https://www.youtube.com/embed/O1qUbrk4Yc8?si=HiSBjmJYhE_oJhB1&amp;start=22"
+    title="YouTube video player"
+    frameborder="0"
+    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+    allowfullscreen
+></iframe>
 
 ## Preparation
 
 To download the latest `docker-compose.yml` open your terminal and navigate to the directory of your choice.
 Run the following command:
 
-```shell
-wget https://goauthentik.io/docker-compose.yml
-```
-
-If this is a fresh authentik installation, you need to generate a password and a secret key. If you don't already have a password generator installed, you can run this command to install **pwgen**, a popular generator:
-
-```shell
-# You can also use openssl instead: `openssl rand -base64 36`
-sudo apt-get install -y pwgen
-```
-
-Next, run the following commands to generate a password and secret key and write them to your `.env` file:
-
-```shell
-echo "PG_PASS=$(pwgen -s 40 1)" >> .env
-echo "AUTHENTIK_SECRET_KEY=$(pwgen -s 50 1)" >> .env
-# Because of a PostgreSQL limitation, only passwords up to 99 chars are supported
-# See https://www.postgresql.org/message-id/09512C4F-8CB9-4021-B455-EF4C4F0D55A0@amazon.com
-```
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+{/* prettier-ignore */}
+<Tabs groupId="OS">
+    <TabItem value="Linux" label="Linux" default>
+        ```shell
+        wget https://goauthentik.io/docker-compose.yml
+        ```
+    </TabItem>
+    <TabItem value="MacOS" label="MacOS">
+        ```shell
+        curl -O https://goauthentik.io/docker-compose.yml
+        ```
+    </TabItem>
+    <TabItem value="Windows" label="Windows">
+        ```shell
+        curl -O https://goauthentik.io/docker-compose.yml
+        ```
+    </TabItem>
+</Tabs>
+
+If this is a fresh authentik installation, you need to generate a password and a secret key. Use a secure password generator of your choice such as pwgen, or you can use `openssl` as below.
+
+Run the following commands to generate a password and secret key and write them to your `.env` file:
+
+{/* prettier-ignore */}
+<Tabs groupId="OS">
+    <TabItem value="Linux" label="Linux" default>
+        ```shell
+        echo "PG_PASS=$(openssl rand -base64 36)" >> .env
+        echo "AUTHENTIK_SECRET_KEY=$(openssl rand -base64 36)" >> .env
+        ```
+    </TabItem>
+    <TabItem value="MacOS" label="MacOS">
+        ```shell
+        echo "PG_PASS=$(openssl rand -base64 36)" >> .env
+        echo "AUTHENTIK_SECRET_KEY=$(openssl rand -base64 36)" >> .env
+        ```
+    </TabItem>
+    <TabItem value="Windows" label="Windows">
```

Comment:
I skipped this part when reviewing, but that implies we support running on Windows, which we probably don't want to.